docs/architecture/cloud-native/candidate-apps.md

```diff
@@ -1,54 +1,88 @@
+Applying a cost/benefit analysis, there's a good chance that most wouldn't support the hefty price tag required to be cloud native. The cost of being cloud native would far exceed the business value of the application.
 
-What types of applications might be candidates for a cloud native approach?
+What type of application might be a candidate for cloud native?
 
-- Large, strategic enterprise systems that need to align to business capabilities/features
+- A large, strategic enterprise system that needs to constantly evolve business capabilities/features
 
-- Systems that require a high release velocity - frequent releases with high confidence
+- An application that requires a high release velocity - with high confidence
 
-- The ability to immediately release features without redeploying the entire system
+- A system with where individual features must release *without* a full redeployment of the entire system
 
-- Applications developed by heterogenous teams with expertise in different technology stacks
+- An application developed by teams with expertise in different technology stacks
 
-- Applications with components that must scale independently
+- An application with components that must scale independently
 
-While we'd all like to maximize our developer excitement level building brand new (that is, green field) systems, often we're charged with modernizing legacy (that is, brownfield) workloads that are critical to the business.
+Then there are legacy systems. While we'd all like to build new applications, we're often responsible for modernizing legacy workloads that are critical to the business. Over time, a legacy application could be decomposed into microservices, containerized, and ultimately "replatformed" into a cloud-native architecture.  
 
-The free Microsoft e-book [Modernize existing .NET applications with Azure cloud and Windows Containers](https://dotnet.microsoft.com/download/thank-you/modernizing-existing-net-apps-ebook) provides a wealth of guidance for migrating on-premises workloads into cloud. Figure 1-11 shows that there isn't a single, one-size-fits-all strategy for migrating legacy applications to the cloud.
+### Modernizing legacy apps
+
+The free Microsoft e-book [Modernize existing .NET applications with Azure cloud and Windows Containers](https://dotnet.microsoft.com/download/thank-you/modernizing-existing-net-apps-ebook) provides guidance for migrating on-premises workloads into cloud. Figure 1-8 shows that there isn't a single, one-size-fits-all strategy for modernizing legacy applications.
 
 ![Strategies for migrating legacy workloads](media/strategies-for-migrating-legacy-workloads.png)
```

Comment:
The other thing related to the expected structure is that this would live inside a folder with the same name as the article like this: 

```suggestion
![Strategies for migrating legacy workloads](./media/definition/strategies-for-migrating-legacy-workloads.png)
```